### PR TITLE
rework tutorial for 3.2

### PR DESCRIPTION
**Pull request by evildmp**

### CODE REVIEW

**Comment by mkoistinen** [comment]
Looking good, @evildmp! Can you install the latest djangocms-admin-style and recapture the screens? (sorry)

**Comment by evildmp** [comment]
There seems to be some incompatibility between the project as created by djangocms-installer and the develop version of django-filer::

    ImportError: cannot import name FILER_STATICMEDIA_PREFIX

**Comment by evildmp** [comment]
That little fight with tracebacks was something I could have done without. Still, here are the updated docs.

**Comment by mkoistinen** [comment]
LGTM!

File: docs/user/tutorial/create_page.rst [hunk 1]
```diff
@@ -2,41 +2,73 @@
...
-   :figwidth: 300
-   :align: right
+.. image:: /user/tutorial/images/welcome.png
+   :alt: The 'Create Page wizard'
+   :width: 400
+   :align: center
 
-This reveals the :ref:`side-frame <side-frame>` for page administration.
+Select *Next*, and provide a *Title* and some basic text content for the new page (you'll be
+able to add formatting to this text in a moment), then hit **Create**.
```
Comment (FinalAngel) [inline docs/user/tutorial/create_page.rst]: as a pro tip: you can also double click the first entry and automatically proceed with the selection

**Comment by FinalAngel** [comment]
nice work! LGTM

**Comment by mkoistinen** [comment]
@evildmp looks like there's a spelling issue.

**Pull request merged**